bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/thing/combined-sink.xml

```diff
@@ -10,6 +10,7 @@
 		</supported-bridge-type-refs>
 		<label>A Pulseaudio Combined Sink</label>
 		<description>represents a group of pulseaudio sinks, which are combined for synchronous audio</description>
+		<semantic-equipment-tag>AudioVisual</semantic-equipment-tag>
```

Comment:
Should be tagged with "Speaker" which is more fitting because this is a group of speakers.